potoq.go

```diff
@@ -27,10 +25,8 @@ func Serve(listener *net.TCPListener) {
 	if err != nil {
 		panic(err)
 	}
-	l4g.LogBufferLength = 4096
-	logw := l4g.NewFormatLogWriter(os.Stdout, l4g.FORMAT_SHORT)
-	l4g.Global = make(l4g.Logger)
-	l4g.Global.AddFilter("stdout", l4g.DEBUG, logw)
+	log.SetLevel(log.DebugLevel)
+	log.SetOutput(os.Stdout)
```

Comment:
To chyba zostawiłbym użytkownikowi do ustawienia.

Reply:
Tak samo ustawienia loggerów w cloudyChat powinny być robione tylko jeżeli są nil. Wystawmy tamte zmienne globalne publicznie i tyle?